src/client/java/be/minelabs/client/mixin/ClientPlayerInteractionManagerMixin.java

```diff
@@ -0,0 +1,31 @@
+package be.minelabs.client.mixin;
+
+import be.minelabs.world.dimension.ModDimensions;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.ClientPlayerInteractionManager;
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(ClientPlayerInteractionManager.class)
+public abstract class ClientPlayerInteractionManagerMixin {
+
+    @Shadow @Final private MinecraftClient client;
+
+    // Note: should stay consistent with variable in ServerPlayNetworkHandlerMixin
+    @Unique
+    private static final float SUBATOMIC_REACH_DISTANCE = 20f;
```

Comment:
Maybe add a Class for this ?
Like the Property Class.
That way you only have 1 variable to change... and fixes the consistency problem

Reply:
Yes good idea! Do we already have a "Settings" or "Constants" class that you know of?

Reply:
The closest we have are the GameRules and BlockProperties class.